ev3dev/core.py

```diff
@@ -2028,6 +2030,15 @@ def driver_name(self):
         return value
 
     @property
+    def auto_mode(self):
+        return True
```

Comment:
Why? I can't imagine we have anyone using this property... I think we should just remove it.

Reply:
We don't use it in any of the demos...I would be ok with chopping it